content/docs/learning-guide/tutorials/remote-repositories/create-remote-repository.md

```diff
@@ -0,0 +1,193 @@
+---
+linktitle: Your First Remote Repository
+title: Your First Remote Repository
+description: Welcome to the Create Remote Repository! Building upon the foundation of Open 3D Engine (O3DE), we have created a new remote repo template, a comprehensive solution for managing and distributing projects effortlessly. You can seamlessly create, package, and share your projects remotely with Github or other version control platforms, promoting collaboration and accelerating project development.
+
+---
+## Add a Remote Repository with the O3DE Command Line Tool
+---
+Navigate to scripts folder from the root of your **Open 3D Engine (O3DE)**.<br>
+
+```
+cd scripts
+```
+Show a list Open 3D Engine (O3DE) commands.
+```
+cd scripts -help
+```
+<strong>Note:</strong> All engine commands have their abbreviations. Use the `-help` menu for list of details.
+
+<br>
+
+## Start With A Github Repository Or Other Version Control Systems
+---
+<ul>
+
+Create a new repository for your **Remote project** and clone this repo to your local 
+machine.
+<br>
+This version control repository link is where you upload your project (`repo-uri`).<br>
+Here is an example of a Github repo uri:
+
+```
+"https://github.com/<YourGitAccount>/<RemoteProjectName>.git"
+```
+</ul>
+<br>
+
+## Create Remote Repository Configuration
+---
+Provide the local path on your machine of your remote project `repo-path` this is where you cloned you online version control repo.<br>
+Note: `--repo-path` and `--repo-uri` are required to create remote repo. 
+<br>
+Use ./o3de.bat if you are using powershell <br>
+```
+o3de.bat create-repo --repo-path "C:\o3de_Projects\<RemoteProjectName>" --repo-uri "https://github.com/<YourGitAccount>/<RemoteProjectName>.git"
+```
```

Comment:
Please use tabs to provide Windows and Linux specific instructions 
```suggestion
{{< tabs name="O3DE CLI command" >}}
{{% tab name="Windows" %}}

```cmd
o3de.bat create-repo --repo-path "C:\o3de_Projects\<RemoteProjectName>" --repo-uri "https://github.com/<YourGitAccount>/<RemoteProjectName>.git"
```
NOTE: Use ./o3de.bat if you are using powershell <br>

{{% /tab %}}
{{% tab name="Linux" %}}

```shell
./o3de.sh create-repo --repo-path "~/<RemoteProjectName>" --repo-uri "https://github.com/<YourGitAccount>/<RemoteProjectName>.git"
```

{{% /tab %}}
{{< /tabs >}}